File: src/main/java/com/adjorno/billib/rest/db/TrackCover.java
package com.adjorno.billib.rest.db;

import javax.persistence.*;

@Entity
@Table(name = "TRACK_COVER")
public class TrackCover {
    @Id
    @Column(name = "TRACK_ID")
    private Long mTrackId;

    @Column(name = "COVER_URL")
    private String mCoverUrl;

    public TrackCover() {
    }

    public TrackCover(Long trackId, String coverUrl) {
        mTrackId = trackId;
        mCoverUrl = coverUrl;
    }

    public Long getTrackId() {
        return mTrackId;
    }

    public void setTrackId(Long trackId) {
        mTrackId = trackId;
    }

    public String getCoverUrl() {
        return mCoverUrl;
    }

    public void setCoverUrl(String coverUrl) {
        mCoverUrl = coverUrl;
    }

    @Override
    public String toString() {
        return mTrackId + " - " + mCoverUrl;
    }
}

File: src/main/java/com/adjorno/billib/rest/db/TrackCover.kt
package com.adjorno.billib.rest.db

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "TRACK_COVER")
data class TrackCover(
        @Id
        @Column(name = "TRACK_ID")
        var trackId: Long? = null,

        @Column(name = "COVER_URL")
        var coverUrl: String? = null
)
